freighter-tests/src/freighterTest/kotlin/freighter/testing/NullHolderOnObserverTest.kt

```diff
@@ -42,32 +43,32 @@ class NullHolderOnObserverTest : DockerRemoteMachineBasedTest() {
 
 	val freighterHelperCordapp = NodeBuilder.DeployedCordapp.fromClassPath("freighter-cordapp-flows")
 
-	@Test(timeout = 300_000)
+	@Test @Timeout(300_000)
```

Comment:
Why do these tests need this new annotation, whilst other tests don't? If we don't need this annotation then  revert.

Reply:
There was compilation failures and probably because from library which is being used has been changed